Sources/Workspace/ResolverPrecomputationProvider.swift

```diff
@@ -95,7 +96,7 @@ private struct LocalPackageContainer: PackageContainer {
     let currentToolsVersion: ToolsVersion
 
     func versionsAscending() throws -> [Version] {
-        if let version = dependency?.state.checkout?.version {
+        if case .version(let version, revision: _) = dependency?.state.checkout {
```

Comment:
please double check logic still looks right

Reply:
Looks correct to me.